RAD-Seq/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Galaxy - RAD-Seq training material
+
+FROM engineson/galaxy-rad-seq
```

Comment:
Can you add this flavor to https://github.com/bgruening/docker-galaxy-stable#list-of-galaxy-flavours--toc 
Looks nice!